package.json

```diff
@@ -29,9 +29,11 @@
   ],
   "types": "./types/index.d.ts",
   "scripts": {
-    "test": "npx jest",
-    "build": "npx tsc && npx webpack",
-    "bench": "npx webpack && node ./bench/bench.js"
+    "build": "tsc && webpack",
+    "prepare": "npm run build",
+    "pretest": "npm run build",
+    "test": "jest",
+    "bench": "npm run build && node ./bench/bench.js"
```

Comment:
This changes two things:

- it adds prepare and pretest scripts
- it changes the `test` and `build`

You explained why the first. Why the second change?

Reply:
`npx` is not required in scripts, it will work.

https://docs.npmjs.com/cli/v9/using-npm/scripts#path